_posts/2020-07-13-our-code-review-etiquette.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: Our Code Review Etiquette
+date: 2020-07-13 09:30:00
+categories: [best-practices]
+author: cleiviane
+---
+
+Code Review is one of the greatest tools we have as software developers to help us improve the quality of our code. It can be incredibly beneficial, but it can also be a source of pain, frustration, and overall, a waste of time instead of a time-saver.
+
+Because of that a while ago we wrote these code review tips that should be acknowledged and incorporated by everyone in our team and now we want to share them with you.
```

Comment:
```suggestion
Because of that, a while ago we wrote these code review tips that should be acknowledged and incorporated by everyone in our team and now we want to share them with you.
```